src/@batch-flask/ui/entity-commands/entity-command.ts

```diff
@@ -17,6 +18,10 @@ export enum EntityCommandNotify {
 }
 
 export interface EntityCommandAttributes<TEntity extends ActionableEntity, TOptions = void> {
+    /**
+     * Name is to be used by feature to show buttons. Or to define keyboard shortcuts
+     */
+    name?: string;
```

Comment:
remove the ? for this one. I added it so it wouldn't break everything but now all commands should have a name(Add to those missing)